composer.json

```diff
@@ -108,7 +108,8 @@
 			"phpunit/phpunit": {
 				"Fix PHP 7 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php7.patch",
 				"Fix PHP 8 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php8.patch",
-				"Fix PHP 8.1 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php81.patch"
+				"Fix PHP 8.1 compatibility": "https://yiisoft.github.io/phpunit-patches/phpunit_php81.patch",
+				"Fix PHP 8.3 compatibility": "tests/phpunit_php83.patch"
```

Comment:
Could be moved to `https://yiisoft.github.io/phpunit-patches/`.

Reply:
Ofcourse, but only if you think this is the right solution.

Reply:
@samdark Could you comment on the above? I can then see how to finish and merge this PR.

Reply:
Yes. I think it is worth moving there.